### PR TITLE
Fix unicode vs bytes error

### PR DESCRIPTION
**Pull request by feerrenrut**
This PR fixes many issues with the translation system, these are documented in the commit history (which we should preserve via regular merge).

Additionally this PR improves the logging for the translation system webhook.

### CODE REVIEW

**Review by michaelDCurran — APPROVED** [review]

**Comment by feerrenrut** [comment]
Some further issues were found, I have pushed some more commits. I have tried to make our regex string type more uniform (for ease of readability), and the same with how we call python from `convertOne.sh`. @michaelDCurran Could you please review this again.

**Review by michaelDCurran — APPROVED** [review]

**Comment by feerrenrut** [comment]
These changes are active on the translation system. I'll wait on confirmation that this fixes the issue before merging.

**Comment by feerrenrut** [comment]
Lots more issues have been fixed. I have been gradually testing and fixing issues by using this branch on the server.

File: hooks/webhook
Comment (michaelDCurran) [inline hooks/webhook]: Should this be commented out?

File: hooks/webhook
Comment (michaelDCurran) [inline hooks/webhook]: How does this happen?

**Review by michaelDCurran — APPROVED** [review]

**Pull request merged**
